package.json

```diff
@@ -3,12 +3,7 @@
   "version": "1.0.0",
   "description": "Chrome Extension of SUSI AI",
   "dependencies": {},
-  "devDependencies": {
```

Comment:
You deleted all dev dependencies which were needed @stealthanthrax you should at least run jshint tests once.